es.json

```diff
@@ -1,71 +1,71 @@
 		},
 		"file": {
-			"name": "File",
-			"option-confirm-file-deletion": "Confirm file deletion",
-			"option-confirm-file-deletion-description": "Whether you want to get prompted to confirm file deletion.",
-			"option-delete-destination": "Deleted files",
-			"option-delete-destination-description": "What to do with files after you delete them.",
-			"option-choice-system-trash": "Move to system trash",
-			"option-choice-vault-trash": "Move to Obsidian trash (.trash folder)",
-			"option-choice-permanent-delete": "Permanent delete",
-			"option-always-update-links": "Always update links when",
-			"option-always-update-links-description": "Whether to automatically update links after renaming a file or prompt.",
-			"option-new-note-location": "New note default location",
-			"option-new-note-location-description": "Where to put newly created notes. Plugin settings will override this.",
-			"option-choice-vault-root": "Vault root",
-			"option-choice-current-folder": "Same folder as the file is in",
-			"option-choice-specified-folder": "In the folder specified below",
-			"option-new-file-folder-path": "Folder to create new notes in",
-			"option-new-file-folder-path-description": "Newly created notes will appear under this folder. Plugin settings will override this.",
-			"option-link-autocompleted-format": "New link format",
-			"option-link-autocompleted-format-description": "What links to insert when auto generating internal links.",
-			"option-choice-shortest-linktext": "Shortest path when possible",
-			"option-choice-relative-path": "Relative path to file",
-			"option-choice-absolute-path": "Absolute path in vault",
-			"option-attachment-folder-path": "Attachment folder path",
-			"option-attachment-folder-path-description": "Newly created attachment files will appear here. That includes images created via drag and drop or pasting, and audio recordings."
+			"name": "Fichero",
+			"option-confirm-file-deletion": "Confirmar borrado de fichero",
+			"option-confirm-file-deletion-description": "Si se desea confirmar el borrado del fichero.",
+			"option-delete-destination": "Ficheros borrados",
+			"option-delete-destination-description": "Qué hacer con los ficheros una vez han sido borrados.",
+			"option-choice-system-trash": "Mover a la papelera del sistema",
+			"option-choice-vault-trash": "Mover a la papelera de Obsidian (carpeta .trash)",
+			"option-choice-permanent-delete": "Borrar permanentemente",
+			"option-always-update-links": "Siempre actualizar enlaces cuando",
+			"option-always-update-links-description": "Si los enlaces se actualizan automaticamente después de renombrar un fichero o preguntar.",
+			"option-new-note-location": "Lugar por defecto para nuevas notas",
+			"option-new-note-location-description": "Dónde poner las nuevas notas.Es sobreescrito por configuraciones de plugin.",
+			"option-choice-vault-root": "Raiz de Caja fuerte",
```

Comment:
Maybe "Vault" shouldn't be translated? Otherwise Bóveda might be more appropriate

Reply:
I agree that caja fuerte doesn't sound totally right though the sense of the word is spot on. I think bóveda is right for Latin American variations as in Spain is more an architectural term (https://dle.rae.es/b%C3%B3veda). 
Maybe we could leave it as "Vault" until they resolve how to do variations?

Reply:
Huh! Interesting! I'm not sure in which content is Obsidian using vault here. Maybe @ericaxu can comment. Otherwise I agree maybe leaving Vault would be best

Reply:
I think of vault (as used in Obsidian) is like a "box" where you store your precious knowledge